README.md

```diff
@@ -428,6 +428,16 @@ a.versions.length                         # 2
 a.paper_trail.previous_version.title      # 'My Title'
 ```
 
+The `:ignore` option can also accept `Hash` arguments that we are considering deprecating.
+
+```ruby
+class Article < ActiveRecord::Base
+  has_paper_trail ignore: [:title, { color: proc { |obj| obj.color == "Yellow" } }]
+end
+```
+
+If an attribute is ignored and only that attribute was updated, then a version won't be created for timestamp attributes.
```

Comment:
I may be misunderstanding, but I think this has already been said, above.

Reply:
Thank you, I missed that 👍